src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -102,15 +115,17 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
     public componentDidUpdate(prevProps: Readonly<IVideoAssetProps>) {
         if (this.props.asset !== prevProps.asset) {
             this.setState({ loaded: false });
+        } else {
+            this.addAssetTimelineTags(this.props.childAssets, this.videoPlayer.current.getState().player.duration);
```

Comment:
Is this the right condition?  It feels like this should only run when `this.props.childAssets` changes?

Reply:
Yeah, this should only update when childAssets changes, when I was doing my testing that was the only thing I was changing, so naturally it was the only thing I saw changing. :)